### PR TITLE
Modifying tests to fix issue with pod creation

### PR DESCRIPTION
**Pull request by JoaoFula**
## Type of change

- [x ] test fix

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by tisnik — COMMENTED** [review]
it looks ok!

**Comment by JoaoFula** [comment]
[OLS-1290](https://issues.redhat.com/browse/OLS-1290)

File: tests/e2e/utils/ols_installer.py [hunk 1]
```diff
@@ -330,8 +330,28 @@ def install_ols() -> tuple[str, str, str]:  # pylint: disable=R0915 # noqa: C901
...
+            ).stdout
+        )
+        == 1,
+    )
+    r = retry_until_timeout_or_success(
+        OC_COMMAND_RETRY_COUNT,
+        5,
+        lambda: len(
+            cluster_utils.run_oc(
+                ["get", "pods", "--field-selector=status.phase=Pending"]
```
Comment (xrajesh) [inline tests/e2e/utils/ols_installer.py:350]: @JoaoFula This is again waiting for `Pending` state ?

Comment (JoaoFula) [reply tests/e2e/utils/ols_installer.py:350]: one wait is for the CreateContainer pod to exist and the second one for it to disappear. It's the comparison to the len that changes.

**Comment by xrajesh** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**